backend/agents/base.py

```
@@ -0,0 +1,46 @@
+import requests
+from rag import recover_context
+from config import OLLAMA_BASE_URL, OLLAMA_LLM_MODEL
+from models import GraphState
+
+def _call_ollama(prompt: str) -> str:
+    """Chama o modelo LLM via Ollama e retorna a resposta."""
+    response = requests.post(
```

Ruff (0.15.5)

[error] Probable use of `requests` call without timeout

(S113)